test/test_runner.go

```diff
@@ -599,6 +602,30 @@ func (r *TestRunner) interpreterContractValueHandler(
 						interpreter.StringStorageMapKey(location.Name),
 					)
 				}
+
+				// We need to store every slab of `blockchainStorage`
+				// to the current environment's storage, so that
+				// we can access fields & types.
+				iterator, err := blockchainStorage.SlabIterator()
+				if err != nil {
+					panic(err)
+				}
+				storage := inter.Storage().(*runtime.Storage)
+
+				for {
+					id, slab := iterator()
+					if id == StorageIDUndefined {
+						break
+					}
+					err := storage.Store(id, slab)
+					if err != nil {
+						panic(err)
+					}
+					err = storage.Commit(inter, true)
+					if err != nil {
+						panic(err)
+					}
```

Comment:
I think this can go outside for (not sure though)

Reply:
It actually makes no difference whether `storage.Commit(inter, true)` is called or not :sweat_smile:
I am quite unsure as to why this is the case. But anyway, I have moved it out of the `for` loop :+1: 
Updated in https://github.com/onflow/cadence-tools/pull/226/commits/ec94c9fcf452697136cb6f6fe6f76e14d3216d07